### PR TITLE
chore: remove repetitive word

### PR DESCRIPTION
**Pull request by acceptacross**

### CODE REVIEW

File: rayon-core/src/lib.rs [hunk 1]
```diff
@@ -228,7 +228,7 @@ type PanicHandler = dyn Fn(Box<dyn Any + Send>) + Send + Sync;
 type StartHandler = dyn Fn(usize) + Send + Sync;
 
 /// The type for a closure that gets invoked when a thread exits. The
-/// closure is passed the index of the thread on which is is invoked.
+/// closure is passed the index of the thread on which is invoked.
```
Comment (cuviper) [inline rayon-core/src/lib.rs:231]: This meant to be "on which **it** is invoked", just like the docs for `StartHandler`.

**Review by cuviper — CHANGES_REQUESTED** [review]

File: rayon-core/src/lib.rs (hunk 1 above)
Comment (acceptacross) [reply rayon-core/src/lib.rs]: Modified.

Please review again

**Comment by cuviper** [comment]
Thanks - can you clean up the commit message? Somehow your sign-off got duplicated, and also this is now correcting rather than removing.

(I might usually just fix that myself, but I hesitate more when a sign-off is involved.)

**Comment by acceptacross** [comment]
> Thanks - can you clean up the commit message? Somehow your sign-off got duplicated, and also this change is now _correcting_ rather than _removing_.
> 
> (I might usually just fix that myself, but I hesitate more when a sign-off is involved.)

Sorry, it was my mistake.
Modified, please review again!

**Review by cuviper — APPROVED** [review]
Thanks!

**Pull request merged**